.azurepipelines/MuDevOpsWrapper.yml

```diff
@@ -19,7 +19,7 @@ resources:
       type: github

```

Comment:
I'm leaving this comment here so you need to resolve it before completing the PR.

I don't think all these commits should be squashed together.  I think the commit history needs to be cleaned up (and some things squashed together) but we should merge this PR as a Rebase and Fast-Forward.